sos/report/plugins/snap.py

```diff
@@ -23,8 +25,56 @@ def setup(self):
         self.add_cmd_output("snap list --all", root_symlink="installed-snaps")
         self.add_cmd_output([
             "snap --version",
-            "snap changes"
```

Comment:
Just noticed this was being removed and re-referenced later. Is the output directly from `snap changes` still desired? If so, change the `exec_cmd()` below for it to `collect_cmd()`, as the latter will both save the output to the archive (like `add_cmd_output()`) _and_ return it (like `exec_cmd()`).

Reply:
ah, interesting, good to know :)

I'll get that modified